Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201305.java

```diff
@@ -143,7 +116,7 @@ public static String extractBirthdate(PRPAMT201306UV02ParameterList params) {
     public static PersonName extractPersonName(PRPAMT201306UV02ParameterList params) {
         LOG.trace("Entering HL7Parser201305.ExtractPersonName method...");
 
-        PersonName personname = new PersonName();
+        PersonName personNameMpiLib = new PersonName();
```

Comment:
Optional: why you rename this variable?  The original name make more sense than personNameMpiLib